index.html

```diff
@@ -32,7 +32,7 @@ <h1 class="from-toolbar">This is Sugar</h1>
   <p class="from-toolbar">In the Sugar Neighborhood view, learners see their
   friends clustered around their current activities; from this view they can
   join each other’s activities.  Sugar collaboration works "peer to
-  peer", in a classroom or under a tree; Sugar does not require the
+  peer," in a classroom or under a tree; Sugar does not require the
```

Comment:
Why? The quotes aren't being used for speech, they are being used because the term is colloquial.